spec/system/visit_service_new_spec.rb

```diff
@@ -42,7 +42,18 @@
       fill_in 'service_entity_id', with: entity_id
       click_button 'Create service'
 
-      expect(page).to have_content 'Entity ID is required'
+      expect(page).to have_content I18n.t('service.errors.missing_entity_id')
+    end
+
+    it 'when entity id has spaces between words' do
+      service_name = 'test component'
```

Comment:
Same comment here - test stripping and failing on whitespace in the middle in two separate tests